### PR TITLE
Update Readme and add Taskfile for setup.

### PR DESCRIPTION
**Pull request by amjustin13**
I have also fixed what seemed to be a typo on nvm.go that prevented me from running tests.

Signed-off-by: Amanda Justiniano-Pagan <amjustin@boro-6.boro.hpdd.intel.com>

### CODE REVIEW

**Comment by amjustin13** [comment]
I am working on adding a few things and still experimenting with the ipmctl lib and the test that is currently in the repo. I will create a new pull request for that work.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,41 @@
-# go-ipmctl
+# Go language bindings for the ipmctl API for managing PMMs
+
 Go bindings for the libipmctl native C lib to enable management of DCPMMs from Go source 
+
+This is a Go interface for [ipmctl](https://github.com/intel/ipmctl) which is also a work in progress.
```
Comment (tanabarr) [inline README.md]: don't think you need "which is also a work in progress"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,41 @@
...
+
+## How to Build
+
+This is a [Go](https://golang.orghttps://golang.org/doc/install)
+project, so the Go development tools are naturally required. We
+recommend the most current Go release available. As of July 2018, the project has been built and tested with Go 1.9.
```
Comment (tanabarr) [inline README.md]: we could update this to go 1.11.5 which is what we are currently testing in CI

File: Taskfile.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    cmds:
+      - task: run-unit-tests
+      - task: build-ipmctl
+
+  run-unit-tests:
+    dir: ipmctl
+    cmds:
+      - go test -v
+
+  build-spdk:
```
Comment (tanabarr) [inline Taskfile.yml]: needs to be changed from spdk?

**Review by tanabarr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (amjustin13) [reply README.md]: Agreed. Will change.

File: README.md (hunk 2 above)
Comment (amjustin13) [reply README.md]: Agreed. Will change.

File: Taskfile.yml (hunk 3 above)
Comment (amjustin13) [reply Taskfile.yml]: Yes it does. Good catch, thank you!

**Review by tanabarr — APPROVED** [review]

**Pull request merged**